cpp/benchmarks/io/nvbench_helpers.hpp

```diff
@@ -28,6 +28,7 @@ enum class data_type : int32_t {
   INTEGRAL        = static_cast<int32_t>(type_group_id::INTEGRAL),
   INTEGRAL_SIGNED = static_cast<int32_t>(type_group_id::INTEGRAL_SIGNED),
   FLOAT           = static_cast<int32_t>(type_group_id::FLOATING_POINT),
+  BOOL8           = static_cast<int32_t>(cudf::type_id::BOOL8),
```

Comment:
It's not a strict requirement, but I find placing BOOL8 before INTEGRAL improves consistency.

Reply:
In the definition of type_id in types.hpp, BOOL8 is listed after all of the ints and floats. I was trying to maintain consistency with that.